serveradmin/settings.example.py

```diff
@@ -15,11 +15,13 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'HOST': 'localhost',
         'NAME': 'serveradmin',
+        'USER': 'serveradmin',
+        'PASSWORD': 'serveradmin',
```

Comment:
Without `HOST` it defaults to the Unix socket connection.  Then the operating system user would be allowed to connection to the database with the same username without a password.

Reply:
Fixed